app-shells/zsh/files/zsh-5.9-musl-E02xtrace-test-fix.patch

```diff
@@ -0,0 +1,24 @@
+# Taken from Alpine Linux, please refer:
```

Comment:
I guess this one is okay for musl only, eve nif I don't love it (https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#conditional-patching).

Is there some way to ask zsh to skip tests? Could we e.g. skip this test (the file maybe even) entirely on musl?

Reply:
I've seen the zsh ebuild skipping some tests by deleting the file, we can try that

Reply:
Yes, please (only for musl though).

Reply:
Something like this ?

```sh
if use elibc_musl ; then
    rm "${S}"/Test/E02xtrace.ztst || die
fi
```

Reply:
Updated